clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -179,18 +179,7 @@ func (w *waitForControlledPodsRunningMeasurement) start() error {
 			w.queue.Add(&f)
 		},
 	)
-
-	go w.informer.Run(w.stopCh)
-	timeoutCh := make(chan struct{})
-	timeoutTimer := time.AfterFunc(informerSyncTimeout, func() {
-		close(timeoutCh)
-	})
-	defer timeoutTimer.Stop()
-	if !cache.WaitForCacheSync(timeoutCh, w.informer.HasSynced) {
-		return fmt.Errorf("timed out waiting for caches to sync")
-	}
-
-	return nil
+	return informer.StartAndSync(w.informer, w.stopCh, informerSyncTimeout)
```

Comment:
We need to be consistent - either you use "w.informer" in all measurements to detect if it's running (then change the measurement above) or you introduce a dedicated isRunning (and then this one has to be changed).

[And potentially other in both options.]

Reply:
I unified it. Now it uses `isRunning` in both metrics.